### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by albertkantwi**
This is a simple Portfolio project. I humbly request for a review. Thank you.

### CODE REVIEW

**Review by skyv26 — COMMENTED** [review]
## Hi @albertkantwi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linters errors. Kindly fixed them all and submit the project for review again from the dashboard.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by tugsem — COMMENTED** [review]
## Invalid Code Review Request ❌ 

Hi @albertkantwi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<h1>You have submitted a project, that doesn't have the right readme template</h2>
Here's  a reminder for you to keep in mind when filling out the template:

> README file should be descriptive:
> * All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present  (**if any relevant header is missing, kindly mark the PR as invalid**)
>   * The only **optional** section is FAQ.
>   * Make sure to add the section **Future Features**, including 1-3 potential features to be added.
> * You should **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with your custom content.
> * You should remove **template logo**. Optionally, you can replace it with your own logo if you have one.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+</head>
+<body>
+    <section class="container">
+        <div class="logo">
+            My Logo
+        </div>
+            <div class="button">
+               <img src="button.png" alt="buttom" width="40px" height="40px"> 
+            </div>
+    </section>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure that the container of the navigation barre section is in the `header` tag. :+1: 

* Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.The container of the section is a header tag.


- Kindly make sure the logo (if the design has one) has a link (`a` tag). :+1: 

   * This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
   
- and also the hamburger button has a `nav` tag.

   * The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. The logo (if the design has one) is on the left and has a link (a tag).

```
eg.   
     
   <header class="container">
        <a href="#"> My Logo</a> 
            <nav class="button">
               <img src="button.png" alt="buttom" width="40px" height="40px"> 
            </nav>
    </header>

```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        LET'S CONNECT
+      </div>
+      <div class="social">
+        <img src="twitter.png" alt="twitter" class="twitter" width="19.99px" height="19.99px">
+        <img src="Linkedin icon@2x.png" alt="linkedin"class="linkedin" wdth="20px" height="20px">
+        <img src="media.png" alt="media" class="media" width="20px" height="16px">
+        <img src="Vector@2x.png" alt="vector" class="vector" width="17px" height="20px">
+        <img src="vecto@2x.png" alt="vecto" class="vecto" width="14px" height="20px">
+      </div>
+    </section>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). :+1: 
   * Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  height: 146px;
+  color: 344563;
+  padding: 8px 24px 0 24px;
+}
+
+.second-container {
+  background-image: url(mobile-background.png);
+  height: 100vh;
+}
+
```
Comment (MaganaAsiati) [inline style.css]: - Kindly make sure that your background image is well  disposed and is not repeated to do that i can suggest you to add that property I listed below to this class :+1: 



```
.second-container {
  background-image: url(mobile-background.png);
  background-repeat: no-repeat;
  background-size: cover;
  min-height: calc(100vh - 599px);
  border-bottom-left-radius: 90px;
  background-color: #fff;
}
```

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @albertkantwi , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      </div> 
+      <div>
+        <p class="smaller-text">I'am a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      </div> 
+      <div class="connect">
+        LET'S CONNECT
+      </div>
+        <ul class="social">
+          <li>
+        <img src="twitter.png" alt="twitter" class="twitter" width="19.99px" height="19.99px">
```
Comment (Kamba56) [inline index.html]: - [ ] As suggested by the previous reviewer, kindly wrap all your social icons in an a tag. This is to improve user experience, when you add the link to your social as a href attribute, users can easily visit your social profile by clicking on the icon.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <title>PortfolioS</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class="container">
+        <a href="#" class="logo">
```
Comment (Kamba56) [inline index.html:11]: - [ ] Great job wrapping the logo in an a tag. By default, a tags come with an underline but from the figma template, the logo isn't underlined. Kindly remove the underline on the logo.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <title>PortfolioS</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class="container">
```
Comment (Kamba56) [inline index.html:10]: - [Optional] Great job using a header element. I noticed that the spaces to the left and right of your header is more than that on the template. You are expected to stick to design guidelines as much as possible and this includes margins and paddings. Kindly fix this please.
- Figma
Screenshot_2023-04-28-15-01-49-466_com android chrome~2
- Your design 
Screenshot_2023-04-28-14-56-57-442_com android chrome~2

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @albertkantwi , 

Great work on making the changes requested by a previous reviewer 👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great job using flexbox to layout contents of your page !👏👏👏
- [x] 👏👏👏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  padding-right: 200px;
+  flex-direction: row-reverse;
+}
```
Comment (ridwanediallo) [inline style.css]: - [ ] Kindly as requested by the previous reviewer ensure that the logo is not underlined to make it similar to the template. :ok_hand: 

### Your logo 
Screenshot from 2023-04-28 16-03-19


### Loog from the template
Screenshot from 2023-04-28 16-03-08

To remove the underline add `text-decoration: none;` to the `logo` class.
```suggestion
.logo {
  display: flex;
  order: 0;
  color: #6070ff;
  width: 76px;
  height: 20px;
  font-family: 'Poppins', sans-serif;
  font-style: normal;
  font-weight: 700;
  font-size: 18px;
  line-height: 20px;
  padding-right: 200px;
  flex-direction: row-reverse;
  text-decoration: none;
}
```

File: style.css [hunk 8]
```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+.container {
+  display: flex;
+  justify-content: space-around;
+  width: 375px;
+  height: 48 px;
+}
```
Comment (ridwanediallo) [inline style.css]: - [ OPTIONAL ] Currently, the logo is not correctly aligned with the menu icon compared to the original design.

To fix that, you only need to add `align-items: center;` to the `container` class.
```suggestion
.container {
  display: flex;
  justify-content: space-around;
  align-items: center;
  width: 375px;
  height: 48 px;
}
```

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @albertkantwi , 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your commit messages are descriptive. ✔


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**